### DISCUSSION TITLE
Brightness keys stop working randomly, than randomly work again

### DISCUSSION THREAD

**Discussion by mickgotwings** [MonitorControl/MonitorControl#949] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

From time to time pressing standard brightness keys stops adjusting brightness. Nothing specific seems to lead to this behavior. Then, after some time, they return to their normal behavior.

Weirdly, when the keyboard brightness keys don't actually change the brightness, you can still change the brightness using the slider in the menu bar.

### Steps to reproduce

1. Use your computer as normal
2. Brightness keys stop working
3. Wait for a couple of hours (I don't know the exact time, it's just an example)
4. It starts working again

### Expected behavior

Brightness is adjustable using brightness keys at all times

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.2.1
- Mac model: Mac mini (M1, 2020)
- MonitorControl version: 4.0.2 Build 6965
- Monitor(s): AOC U34G2G1
- Apple Silicon/M1 (yes or no): yes
```

**Comment by mickgotwings** [comment]
Might be useful as well: neither reconnecting and/or power cycling the monitor, nor restarting the program helped.

**Comment by waydabber** [comment]
If you are using the built-in media keys then it might be that some other apps are interfering (like Karabiner etc)?

I'll convert this one into a discussion, maybe others have better suggestions about this one. :)

**Reply by mickgotwings** [reply]
They're kinda built-in. I'm using a Magic Keyboard. I guess we may call them built-in )
I don't use any other software that might interfere (at least none that would seem to do so).
Are there any logs that would show if MonitorControl gets my input or not?

**Reply by waydabber** [reply]
yeah, I meant Apple keyboard.

Some aspects of the app are logged and can be seen using the console app but this is not really useful for figuring out what is happening with the keyboard. You can download the app and run in XCode though and debug the problem though.

**Comment by mickgotwings** [comment]
A little update: I've been able to bypass the problem by assigning a custom keyboard shortcuts to change the brightness. Seems to me that it's an environment problem, rather than a bug in MonitorControl.

**Reply by waydabber** [reply]
Yes, probably, I think an other app might interfere somehow, but such things are super hard to figure out. :)

**Comment by vanakema** [comment]
The media keys for brightness adjustment aren't working for me either. This is a pretty sudden change, not sure what's going on. This includes immediately after a restart. 

UPDATE: Ok it was Webroot which was technically open in the bg. Weird
UPDATE 2: Ok it was Webroot's "Secure keyboard entry mode"
